spec.html

```diff
@@ -25000,7 +24999,7 @@
         <emu-alg>
           1. Let _x_ be ? thisNumberValue(*this* value).
           1. Let _f_ be ? ToInteger(_fractionDigits_). (If _fractionDigits_ is *undefined*, this step produces the value 0.)
-          1. If _f_ &lt; 0 or _f_ &gt; 20, throw a *RangeError* exception. However, an implementation is permitted to extend the behaviour of `toFixed` for values of _f_ less than 0 or greater than 20. In this case `toFixed` would not necessarily throw *RangeError* for such values.
+          1. If _f_ &lt; -20 or _f_ &gt; 100, throw a *RangeError* exception.
```

Comment:
@littledan , just a note:
You need to change the rest of the `Number#toFixed` to work with negative fractionDigits:

-            1. If _f_ &ne; 0, then
+            1. If _f_ &gt; 0, then
              1. Let _k_ be the number of elements in _m_.
              1. If _k_ &le; _f_, then
                1. Let _z_ be the String consisting of _f_+1-_k_ occurrences of the code unit 0x0030 (DIGIT ZERO).
                1. Let _m_ be the concatenation of Strings _z_ and _m_.
                1. Let _k_ be _f_ + 1.
              1. Let _a_ be the first _k_-_f_ elements of _m_, and let _b_ be the remaining _f_ elements of _m_.
              1. Let _m_ be the concatenation of the three Strings _a_, `"."`, and _b_.
+            1. If _f_ &lt; 0 and _m_ &ne; "0", then
+               1. Let _m_ be the concatenation of the three Strings _m_, the String consisting of 0-_f_ occurrences of the code unit 0x0030 (DIGIT ZERO).
          1. Return the concatenation of the Strings _s_ and _m_.